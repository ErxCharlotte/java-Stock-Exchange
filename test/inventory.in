ADD trader1 100.55
ADD trader2 20.00
IMPORT trader1 ABC 100
IMPORT trader1 CDE 1000
INVENTORY trader1
INVENTORY trader3
INVENTORY trader2
EXIT