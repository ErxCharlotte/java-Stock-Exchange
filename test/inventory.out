$ Success.
$ Success.
$ Trader now has 100 units of ABC.
$ Trader now has 1000 units of CDE.
$ ABC
CDE
$ No such trader in the market.
$ Trader has an empty inventory.
$ Have a nice day.
